google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/CsvOptions.java

```diff
@@ -53,6 +55,7 @@ private Builder(CsvOptions csvOptions) {
       this.allowQuotedNewLines = csvOptions.allowQuotedNewLines;
       this.encoding = csvOptions.encoding;
       this.fieldDelimiter = csvOptions.fieldDelimiter;
+      this.nullMarker = csvOptions.getNullMarker();
```

Comment:
why is this mediated via a function vs everything else using direct access?

Reply:
Was on autopilot. Updated to be more consistent with the rest of the function.